python/indexing/regex_mlir_bindings.py

```diff
@@ -0,0 +1,44 @@
+import re
+import sys
+from os.path import exists
+
+
+def ir_module(fp):
+  with open(fp) as f:
+    f = f.read()
+  if not exists(f"{fp}.bkup"):
+    with open(f"{fp}.bkup", "w") as bkup:
+      bkup.write(f)
+    with open(fp, "w") as ff:
+      res = re.sub(
+          r"^class ((?!PYBIND11_EXPORT).*) {$",
+          r"class PYBIND11_EXPORT \1 {",
+          f,
+          flags=re.MULTILINE,
+      )
+      res = res.replace(
+          "class DefaultingPyMlirContext",
+          "class PYBIND11_EXPORT DefaultingPyMlirContext",
+      )
+      ff.write(res)
```

Comment:
See [this comment](https://github.com/iree-org/iree-llvm-sandbox/pull/687/files#r1166204774).